build.psm1

```diff
@@ -1091,6 +1091,71 @@ function Start-PSBootstrap {
     }
 }
 
+function Start-PSRelease {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [string]$Name, # i.e. v6.0.0-alpha.12
+        [Parameter(Mandatory)]
+        [string]$CommitSHA1 # i.e. 7985366f9ec43af47b24dd88f04c0afc5eab3825
+    )
+
+    Push-Location $PSScriptRoot
+    try {
+        # check that we are in the right location
+        if ($IsWindows) {
+            if ($PSScriptRoot -ne "C:\PowerShell") {
```

Comment:
This is to make `pdb` paths uniformed. See https://github.com/PowerShell/PowerShell/blob/172b2234fc5a82df0350cddeb6f91837b7999225/docs/maintainers/releasing.md#preparing
For the same reason, we don't want any divergancy from `C:\PowerShell`